### DISCUSSION TITLE
用java script來創造DC機器人

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#469] [Show and tell]
[https://b-l-u-e-b-e-r-r-y.github.io/post/DiscordBot01/](url)
在剛開始接觸很多社群像是玩虛擬貨幣的，或者是一些教授開的討論群，其實我剛始一直不知道為什麼非得用DC不可?
LINE群不也挺好用的嗎?後來又發現DC群裡面的機器人剛開始不以為意就覺得跟LINE種官方網站的機器人一樣很好設置，
直到在滑網站文章的時候，偶然看到DC機器人用JAVA來編輯，才了解到DC機器人的不簡單，看一個人怎麼利用就能有很多的功能，
也讓我意識到為何大家都是用DC居多，畢竟能用程式編寫自己要用的功能可方便多了，文中有提及到防止TOKEN上傳到GITHUB的方法 ，一不小心上傳廣告真的就會多超多，自己就有被廣告洗過...超煩的。文章還有教大家如何設置初始命令，蠻實用的。

**Comment by xkllkx** [comment]
謝謝樓主的分享，還蠻實用的！！！

**Comment by jason79461385** [comment]
自己想要做一個音樂機器人呢，感謝分享，暑假有空來試試看。

**Reply by Rosancl** [reply]
想做音樂機器人+1

**Comment by treeann** [comment]
謝謝樓主的分享，還蠻實用的！！棒

**Comment by liangyinglly** [comment]
連結已經點不進去了 好難過
真的好想知道你分享的資訊

**Comment by leon10456** [comment]
謝謝樓主的分享

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by Lynn12378** [comment]
原來DC還有這樣的優點，感謝分享

**Comment by Jimmy911218** [comment]
連結不能用，不過聽起來很不錯哀

**Comment by yenwei12138** [comment]
DC機器人真的很好用，打遊戲做報告時都用來撥音樂

**Comment by Hchejxnc** [comment]
DC的機器人能做的事情真的非常多，像粉絲群的機器人就可以幫忙設置身分組，或是當有什麼最新消息時跳出通知給你
完全看自己怎麼運用~
